docs/user_manual.md

```diff
@@ -226,6 +234,13 @@ If is specified, Helios will unregister from service discovery and wait the
 specified number of seconds before undeploying. Optional, defaults to `0` for
 no grace period.
 
+#### rolloutOptions
+If specified, Helios will use these options when performing a `rolling-update`.
+The order of precedence for these options is as follows.
+1. Values passed in via `rolling-update`'s switches
+2. Values here
```

Comment:
could this be more specific, and say "Values specified in the job object"? "here" was confusing to me at first

Reply:
This is how I worded it in the internal document:

> When a deployment is issued to a helios master, the RolloutOptions will be constructed from the request parameters and job configuration with the following precedence:
>
> 1. Parameters included in the API request (eg, flags passed on the command line)
> 2. Parameters defined in the job configuration (of the job being rolled out)
> 3. The default value for the parameter as defined in the RolloutOptions class

Reply:
Thanks I'll use that.